migrations/sqls/20190806211241-client-interaction-up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS clientInteractions (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    clientInteractionType ENUM (
+        'clientInteractionCreated',
+        'clientInteractionUpdated',
+        'clientInteractionDeleted'
+    ) NOT NULL,
+    clientId INT NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    serviceId INT NOT NULL,
+    interactionType ENUM (
+        'in pesron',
+        'by phone',
+        'workshop/talk',
+        'one on one/light touch',
+        'consultation'
+    ) NOT NULL,
+    description MEDIUMTEXT,
+    date DATE DEFAULT CURRENT_DATE NOT NULL,
+    duration TIME DEFAULT '0:00:00' NOT NULL,
+    location ENUM (
+        'CU Office',
+        'Consulate Office',
+        'Community Event'
+    ) NOT NULL,
+    canModify BOOLEAN DEFAULT false NOT NULL,
+    isDeleted BOOLEAN DEFAULT false NOT NULL,
+    dateAdded DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    addedBy int NOT NULL,
```

Comment:
Let's add two more columns for last update. See linked examples of how to implement it:

https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/9c8ec0fff60ef58fa220283e5abb9c4b925c14c4/migrations/sqls/20190508233238-clients-up.sql#L9

https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/9c8ec0fff60ef58fa220283e5abb9c4b925c14c4/migrations/sqls/20190508233238-clients-up.sql#L11

https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/9c8ec0fff60ef58fa220283e5abb9c4b925c14c4/migrations/sqls/20190508233238-clients-up.sql#L13